doc/R.nvim.txt

```diff
@@ -1744,7 +1744,7 @@ will consider that the browser was closed and will to open it again.
 For Quarto documents, the available key bindings are <LocalLeader>qr (render
 document), <LocalLeader>qp (preview document), and <LocalLeader>qs (stop
 previewing). The command <LocalLeader>kr achieves basically the same result of
-<LocalLeader>kr. The difference is that the generated document is open after
+<LocalLeader>qr. The difference is that the generated document is open after
```

Comment:
For me `kr` works but does not open the generated html file from this qmd document

````
---
params:
  aa: 1
---

Text here.

```{r}
print(params$aa)
```
````

Reply:
On my side, it sends the `nvim.interlace.rmd()` command to R and opens the HTML document in Firefox. What is the value of the "browser" option in your R installation? Does `browseURL("doc_name.html")` works?